src/components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import DboardJumbotronContainer from './jumbotron/DboardJumbotronContainer';
+import UpcomingTripsContainer from './UpcomingTripsContainer';
+import PreviousTripsContainer from './PreviousTripsContainer';
+import DashboardNavContainer from './DashboardNavContainer';
+import './Dashboard.css';
+
+/**
+ *  Dashboard is a component that holds all of the dashboard helper components
+ */
+
+export default class Dashboard extends Component {
+  /**************************** Visual component ****************************/
+  render() {
+    return (
+      <div>
+        <div className="sidebar">
+          <DashboardNavContainer />
+        </div>
+        <div className="container-right">
+          <h1>Hello, {this.props.user.f_name}</h1>
+          <DboardJumbotronContainer />
+          <UpcomingTripsContainer />
+          <PreviousTripsContainer />
```

Comment:
For stylistic purposes, it would be nice if these elements were distinguished from each other a bit more. [Bootstrap cards](https://react-bootstrap.github.io/components/cards/) may be of good use here. (Ignore this comment if this is something you were already planning to do.)

Reply:
Stylistically as in displayed to the user? In that case yupp, Bootstrap cards are coming to them! I'm just going to do a CSS branch and PR next (after #177) so it will be in that! The styling that is in this PR is just what I frantically threw together for the presentation

Stylistically as in the code? Cause in that case I disagree, it seems like unnecessary bulk since the cards (or divs or however you break them up) would be really bulky / cluttered for no real benefit...

Reply:
I mean stylistically visually.